code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java

```diff
@@ -0,0 +1,61 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import com.onlinecontacttracing.storage.PotentialContact;
+import static com.googlecode.objectify.ObjectifyService.ofy;
+import com.onlinecontacttracing.storage.PositiveUser;
+import java.util.Optional;
+import java.util.ArrayList;
+
+@WebServlet("/get-positive-user-info")
+public class PositiveUserInfoServlet extends CheckForApiAuthorizationServlet {
+
+  String getServletURIName() {
+    return "/get-positive-user-info";
+  }
+  
+  
+  /*
+  * This servlet will retrieve data from the People Api
+  * Additionaly it will forward the request to the servlet for Calendar Api
+  * Once both are done, the servlet will merge contact data sets
+  */
+  void useCredential(String userId, Credential credential, HttpServletResponse response) throws IOException, InterruptedException {
+    // Execute runnable to get people data
+    ArrayList<PotentialContact> contactsFromPeople = new ArrayList<PotentialContact>();
+    Thread peopleInfo = new Thread(new PeopleDataForPositiveUser(credential, contactsFromPeople));
+    Thread contactInfo = new Thread(new CalendarDataForPositiveUser(credential));
+
+    peopleInfo.start();
+    contactInfo.start();
+
+    
+    peopleInfo.join();
+    contactInfo.join();
+
+    // Load PositiveUserContacts from objectify
```

Comment:
So, this is just a comment without any code. Maybe you intend to put something here? If so, mark it with TODO(...). Else, get rid of this comment.

Reply:
Added TODO to every comment that needs it